glide.lock

```diff
@@ -22,6 +22,37 @@ imports:
   version: fa29b1d70f0beaddd4c7021607cc3c3be8ce94b8
   subpackages:
   - capnslog
+- name: github.com/docker/distribution
+  version: 82609180a1744d5381cc6ba5c3286d0699c3dccd
+  subpackages:
+  - reference
+  - digest
+- name: github.com/docker/engine-api
+  version: 8d8fffdf863b12d03c76abf6ca1377e6f8f4e549
+  subpackages:
+  - client
+  - types
+  - client/transport
+  - client/transport/cancellable
+  - types/container
+  - types/filters
+  - types/network
+  - types/reference
+  - types/registry
+  - types/swarm
+  - types/time
+  - types/versions
+  - types/mount
+  - types/blkiodev
+  - types/strslice
+- name: github.com/docker/go-connections
```

Comment:
I don't see this in use else where in this PR. Was it just not added some other time when it should have been?